core/scripts/common/helpers.go

```diff
@@ -452,3 +483,38 @@ func IsAvaxNetwork(chainID int64) bool {
 		chainID == 335 || // DFK testnet
 		chainID == 53935 // DFK mainnet
 }
+
+// IsPolygonEdgeNetwork returns true if the given chain ID corresponds to an Pologyon Edge network.
+func IsPolygonEdgeNetwork(chainID int64) bool {
+	return chainID == 100 || // Nexon test supernet
+		chainID == 500 // Nexon test supernet
+}
+
+func CalculateLatestBlockHeader(env Environment, blockNumberInput int) (err error) {
+	blockNumber := uint64(blockNumberInput)
+	if blockNumberInput == -1 {
+		blockNumber, err = env.Ec.BlockNumber(context.Background())
+		if err != nil {
+			return fmt.Errorf("failed to fetch latest block: %+v", err)
+		}
+	}
+
+	blockNumber = blockNumber - 1
```

Comment:
do we have to subtract 1 here? maybe a comment would be helpful

Reply:
oh I see that It validates blockhash below

Reply:
GetRLPHeaders method **increments** the blockNum sent by 1 and then fetches block headers for the child block. So, in order to get the block header for a given block, I had to decrement by 1. 

This is to give a user-friendly command like: `go run ./vrfv2plus/testnet latest-block-header --block-number 453551`